javascript-carousel/main.js

```diff
@@ -0,0 +1,66 @@
+var counter = 1;
+var intervalId = setInterval(switchImageTimer, 3000);
+switchImageTimer();
+
+function switchImageTimer() {
+  // debugger;
+  var image = document.querySelectorAll('img');
+  var circle = document.querySelectorAll('.fa-circle');
+  if (counter <= 5 && counter > 0) {
+    for (var i = 0; i < image.length; i++) {
+      if (counter === parseInt(image[i].getAttribute('image-view'))) {
+        image[i].setAttribute('class', '');
+        circle[i].setAttribute('class', 'fas fa-circle padding-left-right');
+      } else {
+        image[i].setAttribute('class', 'hidden');
+        circle[i].setAttribute('class', 'far fa-circle padding-left-right');
+      }
+    }
+
+  } else if (counter > 5) {
+    image[0].setAttribute('class', '');
+    image[image.length - 1].classList.toggle('hidden');
+    circle[0].setAttribute('class', 'fas fa-circle padding-left-right');
+    circle[image.length - 1].setAttribute('class', 'far fa-circle padding-left-right');
+    counter = 1;
+  } else if (counter === 0) {
+    image[0].setAttribute('class', 'hidden');
+    image[image.length - 1].setAttribute('class', '');
+    circle[0].setAttribute('class', 'far fa-circle padding-left-right');
+    circle[image.length - 1].setAttribute('class', 'fas fa-circle padding-left-right');
+    counter = 5;
+  }
```

Comment:
If you do these conditional checks first to correct what the `counter` value should be, you should be able to do the same work in all cases that you did above with the `for` loop